sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1296,6 +1296,10 @@ public partial class BlobBaseClient
         public virtual System.Uri GenerateSasUri(Azure.Storage.Sas.BlobSasBuilder builder) { throw null; }
         public virtual System.Uri GenerateSasUri(Azure.Storage.Sas.BlobSasPermissions permissions, System.DateTimeOffset expiresOn) { throw null; }
         protected internal virtual Azure.Storage.Blobs.Specialized.BlobLeaseClient GetBlobLeaseClientCore(string leaseId) { throw null; }
+        public virtual Azure.Pageable<Azure.Storage.Blobs.Models.BlobItem> GetBlobSnapshots(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
I went with `BlobBaseClient.GetBlobVersions()` and `.GetBlobSnapshots()`.  We could have also gone with `GetVersions()` and `GetSnapshots()`, but I wanted to be more specific.

Reply:
We don't include `Blob` in other API names (other than getting clients)